### DISCUSSION TITLE
Difference between 'pt' and 'ft'

### DISCUSSION THREAD

**Discussion by bryanbocao** [open-mmlab/mmclassification#1646] [Q&A]
I see 'pt' and 'ft' in different config files, may I know the differences:
such as ```vit-base-p16_ft-64xb64_in1k-384.py``` or ```vit-base-p16_pt-64xb64_in1k-224.py```

https://github.com/open-mmlab/mmpretrain/tree/master/configs/vision_transformer

**Comment by guneetmutreja** [comment]
One is for pretraining and other for finetuning

**Comment by bryanbocao** [comment]
@guneetmutreja Thanks for answering the question! Suppose I train the model from scratch. I think each of them doesn't matter?
